setup.py

```diff
@@ -2,11 +2,12 @@
 
 NAME = "sqlalchemy-kusto"
 DESCRIPTION = "Azure Data Explorer (Kusto) dialect for SQLAlchemy"
-VERSION = "1.1.0"
+VERSION = "2.0.0"
 
 REQUIREMENTS = [
-    "azure-kusto-data==2.1.1",
-    "sqlalchemy==1.3.24",
```

Comment:
Do you want to strictly fix sqlalchemy version?

Reply:
It is a good question. For now, I fixed it to keep the same version as in the apache/superset. I hope it will allow us to avoid unpredictable errors. But we also could make it 1.4.x. What do you think?

Reply:
I prefer the second option - to make it 1.4.x. And fix errors if they occur)

Reply:
Fixed. Please take a look.

Reply:
@xneg, I also added the section "Using with Apache Superset" to the README. I think now it is important to clarify which version of the package you need to use with different Apache Superset versions.

Reply:
That's great! I need first to check its usage with Superset and then I'll approve this PR.
Great work, by the way!